tests/keras/layers/wrappers_test.py

```diff
@@ -97,7 +97,7 @@ def test_TimeDistributed_learning_phase():
     y = wrappers.TimeDistributed(core.Dropout(.999))(x, training=True)
     model = Model(x, y)
     y = model.predict(np.random.random((10, 3, 2)))
-    assert_allclose(0., y, atol=1e-1, rtol=1e-1)
+    assert_allclose(y, np.zeros(y.shape), atol=1e-1, rtol=1e-1)
```

Comment:
The order convention is `expected, actual`. Changing the shape does not affect flakiness (or anything else) since numpy automatically broadcasts. To make the test less flaky you can either increase the dropout or reduce the tolerance.